inference-engine/src/transformations/src/transformations/common_optimizations/normalize_l2_fusion.cpp

```diff
@@ -25,14 +25,17 @@ ngraph::pass::NormalizeL2FusionWithMax::NormalizeL2FusionWithMax() {
     auto pow = std::make_shared<ngraph::opset4::Power>(input, exp);
     auto axes = ngraph::pattern::wrap_type<ngraph::opset4::Constant>();
     auto reduce_sum = std::make_shared<ngraph::opset4::ReduceSum>(pow, axes);
-    auto sqrt = std::make_shared<ngraph::opset4::Sqrt>(reduce_sum);
     auto eps_const = ngraph::pattern::wrap_type<ngraph::opset4::Constant>();
-    auto sqrt_max_eps = std::make_shared<ngraph::opset4::Maximum>(sqrt, eps_const);
-    auto divide = std::make_shared<ngraph::opset4::Divide>(input, sqrt_max_eps);
+    auto max = std::make_shared<ngraph::opset4::Maximum>(reduce_sum, eps_const);
+    auto sqrt = std::make_shared<ngraph::opset4::Sqrt>(max);
+    auto divide = std::make_shared<ngraph::opset4::Divide>(input, sqrt);
```

Comment:
Update or add new tests.

Reply:
I've updated the tests here inference-engine/tests/functional/inference_engine/transformations/normalize_l2_fusion_test.cpp